fonts.css

```diff
@@ -0,0 +1,13 @@
+@font-face {
+	font-family: 'GothaPro';
+	src: url(fonts/GothamPro.woff2) format('woff2'), url('fonts/GothaProReg.woff') format('woff');
+	font-weight: normal;
+	font-style: normal;
+}
+
+@font-face {
+	font-family: 'GothaPro';
+	src: url(fonts/GothamPro-Bold.woff2) format('woff2'), url('fonts/GothaProBol.woff') format('woff');
+	font-weight: 700;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/GerKateAxion/pullRequest?prid=5991795)